packages/@aws-cdk/aws-lambda-python/lib/bundling.ts

```diff
@@ -86,8 +86,8 @@ export class Bundling implements CdkBundlingOptions {
     bundlingCommands.push(packaging.exportCommand ?? '');
     if (packaging.dependenciesFile) {
       bundlingCommands.push(`python -m pip install -r ${DependenciesFile.PIP} -t ${options.outputDir}`);
-    };
-    bundlingCommands.push(`cp -R ${options.inputDir} ${options.outputDir}`);
+    }
+    bundlingCommands.push(`cp -R ${options.inputDir}/* ${options.outputDir}`);
```

Comment:
I believe this syntax will drop hidden files, whereas `cp -R input/ output` will retain them.

Reply:
Ok, let me test that.

Reply:
Updated.